packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss

```diff
@@ -1,16 +1,32 @@
 @use "sass:math";
-
-@use "../../variables/project-alerts" as *;
 @use "../../settings" as *;
 @use "../../functions" as *;
-@use "./at-media" as *;
 @use "./border-box-sizing" as *;
 @use "./set-text-and-bg" as *;
 @use "../general/icon" as *;
 @use "../utilities" as *;
 @use "../typography/typeset.scss" as *;
 @use "../helpers/set-link-from-bg" as *;
 
+$alert-icon-gap: 1.5;
```

Comment:
> [!Note]
> This is the gap between the icon and the text. The gap is currently the same regardless of icon size. From a design perspective, it might be beneficial to adjust this size proportionally to the icon size. I personally have decided the gap looks reasonable as is across variants, but flagging in case it bothers anyone else.